### PR TITLE
Tidy annotations & attribution

### PR DESCRIPTION
**Pull request by nicholascar**
This PR contains removal of duplicate properties (that can be created with inference, if needed), modernisation of DCTERMS removal of attribution per-element and modern, W3C-style, attribution at the whole-of-ontology level.

I have only included me and @jabhay in the contributors field now to show off the style of contributor indication used (schema.org) and to ask for your initial reviews of this and your attribution details before completing this PR.

@jabhay can you please supply OGC direction on attribution?

### CODE REVIEW

File: 1.1/geo.ttl [hunk 1]
```diff
@@ -1,724 +1,492 @@
...
-:gmlLiteral a rdfs:Datatype ;
-	dc:contributor "Matthew Perry" ;
-	dc:creator "OGC GeoSPARQL 1.0 Standard Working Group" ;
-	dc:date "2011-06-16"^^xsd:date ;
-	dc:description """A GML serialization of a geometry object."""@en ;
-	rdfs:comment """A GML serialization of a geometry object."""@en ;
-	rdfs:isDefinedBy <http://www.opengis.net/ont/geosparql> , <http://www.opengis.net/spec/geosparql/1.0> ;
-	rdfs:label "GML Literal"@en ;
+# http://www.opengis.net/ont/geosparql#gmlLiteral
+:gmlLiteral
```
Comment (dr-shorthair) [inline 1.1/geo.ttl:48]: Is it appropriate to remove the existing micro-credits - Matt Perry, original date, etc?

File: 1.1/geo.ttl [hunk 2]
```diff
@@ -1,724 +1,492 @@
...
 
-:geoJSONLiteral a rdfs:Datatype ;
-	dc:contributor "Timo Homburg" ;
-	dc:creator "OGC GeoSPARQL 2.0 Standard Working Group" ;
-	dc:date "2020-10-30"^^xsd:date ;
-	dc:description """A GeoJSON serialization of a geometry object."""@en ;
-	rdfs:comment """A GeoJSON serialization of a geometry object."""@en ;
+# http://www.opengis.net/ont/geosparql#geoJSONLiteral
+:geoJSONLiteral
+	a rdfs:Datatype ;
```
Comment (dr-shorthair) [inline 1.1/geo.ttl:65]: Microcredit for Timo Homburg should be preserved.

**Comment by mperry455** [comment]
@dr-shorthair I think it's a good idea to roll all those microcredits up into one annotation at the ontology level for 1.1. They are quite redundant.

**Comment by dr-shorthair** [comment]
OK - I surmised that might have been going on, but wanted to check.

**Comment by dr-shorthair** [comment]
I could have read @nicholascar comment of course where he kinda says that ...

**Review by situx — APPROVED** [review]
I added my credentials and hereby approve this change request

**Review by dr-shorthair — APPROVED** [review]

**Pull request merged**